sbt-plugin/src/sbt-test/gen-scala-server/01-gen-basic-server/src/main/protobuf/helloworld.proto

```diff
@@ -1,21 +1,21 @@
 syntax = "proto3";
 
 option java_multiple_files = true;
-option java_package = "io.akka.grpc";
+option java_package = "io.grpc.examples.helloworld";
```

Comment:
In Akka we always put the generated proto sources in a `akka.bbb.ccc.proto` should we recommend that here as well (but a `.grpc` ) to discern handwritten classes from generated ones?

Reply:
We can recommend that, but not religiously :)

Reply:
so something like `example.myapp.helloworld.proto` and `example.myapp.echo.proto`?

Reply:
I'd say `example.myapp.helloworld.grpc`, doing that in samples isn't really religiously enforcing anything, is it?

Reply:
I dont mind, as long as wording mentions that packages are a bit up to personal preference, but that's a default you can stick to